versions/1.0/index.md

```diff
@@ -426,36 +426,46 @@ An `Error` **MAY** have the following:
 * `status_code`: Represents the HTTP status code associated with response as integer.
 * `details`: Array of [error detail](#document-components-error-detail).
 
+
+## <a href="#document-components-error-detail" id="document-components-error-detail" class="headerlink"></a> Error Detail
+
+Some errors require more information about what failed and sometimes where the error occurred. API authors can then provide a [node](#document-components-node) of `ErrorDetail` to help pin-point specific errors.
+
+An `ErrorDetail` **MUST** have the following:
+
+* `@type`: Have a value of `ErrorDetail`.
+* `description`: Detail description about the error as string.
+
+An `ErrorDetail` **MAY** have the following:
+* `@links`: Repesents a [link](#document-components-link-collection) with navigational _links_ to other APIs and additional resources.
+* `source`: Represents a JSON Pointer [[RFC6901](https://tools.ietf.org/html/rfc6901)] as string.
+
+
 ```json
 {
     "@type": "Error",
     "code": "invalid_request",
     "status_code": 400,
-    "title": "One or more properties were empty",
-    "description": "One or more required fields were empty or missing from the request.",
+    "title": "One or more properties were empty or invalid",
+    "description": "One or more required fields were empty or not meeting valdiation requirements.",
     "details": [
         {
             "@type": "ErrorDetail",
-            "source": "/given_name",
-            "description": "Must not be empty"
+            "source": "/first_name",
+            "description": "Must not be empty and a mininum of 4 characters"
         },
         {
             "@type": "ErrorDetail",
-            "source": "/family_name",
+            "@link": {
+                "account": {
+                    "href": "/v1/account",
+                    "base_path": "https://api.xyz.co",
```

Comment:
This looks like a host not a base_path

Reply:
That matches the spec on the main hyperion page.

Reply:
@thehelix112 We should square away the language in the `Sub-service Path` section, mixing host and path.